client/my-sites/hosting/sftp-card/ssh-key-loading-skeleton.tsx

```diff
@@ -0,0 +1,38 @@
+import { LoadingPlaceholder } from '@automattic/components';
+import styled from '@emotion/styled';
+
+const PlaceholderGroup = styled.div( {
+	display: 'flex',
+	flexDirection: 'column',
+	gap: '0.5em',
+} );
+
+const ParagraphPlaceholder = styled( LoadingPlaceholder )( {
+	width: 560,
+	maxWidth: '100%',
+} );
+const LinkPlaceholder = styled( ParagraphPlaceholder )( { width: 290, marginBottom: 30 } );
+const LabelPlaceholder = styled( ParagraphPlaceholder )( { width: 50 } );
+const InputPlaceholder = styled( ParagraphPlaceholder )( {
+	width: '80%',
+	height: 36,
+	marginBottom: 20,
+} );
+
+export function SshKeyLoadingSkeleton() {
+	return (
+		<PlaceholderGroup>
```

Comment:
I wonder if we should do a more generic skeleton layout that sits in between the two different states (no SFTP credentials and SFTP credentials)?


https://user-images.githubusercontent.com/36432/222419050-56628f4f-1537-4ea8-96ff-c64ffd50ec89.mp4

Reply:
I like the current placeholder, happy to tweak it if someone has a stronger opinion.
A generic placeholder wouldn't match any of our layouts. So I don't see any problem with our current one.

Reply:
One suggestion would be remove one of the fake inputs+label so it will fit better both designs 👌

Reply:
@sejas I think that works for me! I'm not strongly opinionated about it.

Reply:
Done https://github.com/Automattic/wp-calypso/pull/73982/commits/c0ea4a1a5eec7aa0924f8c79d957d4d1d90e0851